src/app/modules/shared-components/components/progress-level/progress-level.component.ts

```diff
@@ -9,13 +9,13 @@ export class ProgressLevelComponent {
 
   @Input() size = 12;
   @Input() theme : 'default' | 'revert' = 'default';
-  @Input() filled = 0;
-  @Input() total = 4;
+  @Input() value = 0;
+  @Input() maximum = 4;
 
-  constructor() {}
+  totalArray : number[] = [];
```

Comment:
Detail: No need to put an initial value as you initial value in the constructor.
You should add explaining what this strange variable is.
```suggestion
  totalArray : number[]; // array containing [0, 1, 2, ...] as the template cannot iterate by itself
```

Reply:
You just have "commit suggestion" directly in GH.